pandas/core/ops.py

```diff
@@ -1638,6 +1638,7 @@ def dispatch_to_series(left, right, func):
     """
     # Note: we use iloc to access columns for compat with cases
     #       with non-unique columns.
+    right = lib.item_from_zerodim(right)
```

Comment:
is there a reason you needed to add this? (e.g. do we actually hit this in the new code path and not in the original)?

Reply:
Yes.  This PR has `DataFrame._combine_const` send zero-dim arrays down this path, which without this line don't get handled correctly.

Reply:
ok great